src/components/data-processing/CellSizeDistribution/CalculationConfig.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  Space,
+  InputNumber,
+  Form,
+  Button,
+  Alert,
+  Radio,
+} from 'antd';
+
+import _ from 'lodash';
+
+import BandwidthOrBinstep from '../ReadAlignment/PlotStyleMisc';
+
+import { updateProcessingSettings } from '../../../redux/actions/experimentSettings';
+
+const CalculationConfig = (props) => {
+  const {
+    experimentId, sampleId, plotType, sampleIds,
+  } = props;
+
+  const config = useSelector(
+    (state) => state.experimentSettings.processing.cellSizeDistribution[sampleId]?.filterSettings
+      || state.experimentSettings.processing.cellSizeDistribution.filterSettings,
+  );
+
+  const FILTER_UUID = 'cellSizeDistribution';
+
+  const dispatch = useDispatch();
+
+  const [individualChangesWarningEnabled, setIndividualChangesWarningEnabled] = useState(false);
```

Comment:
`Enabled` sounds like a settings that can be enabled/disabled. Would renaming this to `displayIndividualChangesWarning` capture the intent better?